plugin/pkg/admission/podnodeselector/admission.go

```diff
@@ -148,7 +148,7 @@ func (p *Plugin) Validate(ctx context.Context, a admission.Attributes, o admissi
 	if err != nil {
 		return err
 	}
-	if !labels.AreLabelsInWhiteList(pod.Spec.NodeSelector, whitelist) {
```

Comment:
please remove the original.  I don't think we need it anymore and that's a clear signal to everyone they should re-examine their code.

Reply:
ok. i will remove it.